lib/src/commands/test/test.dart

```diff
@@ -57,6 +57,11 @@ class TestCommand extends Command<int> {
         defaultsTo: true,
         help: 'Whether to apply optimizations for test performance.',
       )
+      ..addFlag(
+        'machine',
```

Comment:
I'd prefer to keep the options consistent with the flutter tool as much as possible. In this case rather than `--machine` can we stick to `--reporter json`?

Reply:
Updated it there, but I'm having trouble testing locally, from a clone of the repo I can't get it building/testing successfully. I'm using `dart run bin/very_good.dart test -h` but when it comes to running the tests for this repo, it's not succeeding. Any ideas? Dependencies installed and all, my flutter is 3.0.5, anything else it could be missing?

Reply:
I don't think these changes are enough unfortunately. We're already using the flag internally but we would need to adjust the output based on whether the user explicitly requested a custom reporter.